src/jspdf.js

```diff
@@ -3537,6 +3538,10 @@ function jsPDF(options) {
       }
     }
 
+    if (!renderMaxWidthOverflow) {
+      text = [text[0]];
+    }
```

Comment:
Instead of adding an option to the text method, I think we can call splitTextToSize already in context2d and pass only the first line to `text()`. Also, don't pass `maxWidth` then.